var/spack/repos/builtin/packages/ctffind/package.py

```diff
@@ -34,6 +34,7 @@ def url_for_version(self, version):
 
     patch("configure.patch", when="@4.1.8")
     patch("power9.patch", when="@4.1.14 target=power9le")
+    patch("fix_return_types.patch", when="@4.1.13:4.1.14")
```

Comment:
I see in the comments that fixing this upstream may take a while.  Thanks for pasting the link.

Reply:
Yeah, that fix have been around since 2021 and no fixed official version has been released.